src/cocotb/handle.py

```diff
@@ -151,26 +164,54 @@ def __repr__(self) -> str:
         return type(self).__qualname__ + "(" + desc + ")"
 
 
-IndexType = TypeVar("IndexType")
+KeyType = TypeVar("KeyType")
```

Comment:
Should we add this to ``library_reference.rst``?

Reply:
It's not something we want to be public. I've wondered about how to handle TypVars in docs before a couple years ago. Back then Sphinx did *not* handle TypeVars very well or have any idea what to do with them. Maybe they do now?